data/mysql/schema.sql

```diff
@@ -1897,6 +1897,7 @@ CREATE TABLE `ezsearch_object_word_link` (
   `published` int(11) NOT NULL DEFAULT '0',
   `section_id` int(11) NOT NULL DEFAULT '0',
   `word_id` int(11) NOT NULL DEFAULT '0',
+  `language_mask` bigint NOT NULL DEFAULT 0,
```

Comment:
This is a BC break, will crash Legacy setups on Legacy Bridge as those don't follow eZ Platform upgrade process AFAIK.

Reply:
I'm not sure about SQL dumps in EZ. Could you propose something here @alongosz ?